apps/docs/src/.vitepress/components/ComponentRoadmap.vue

```diff
@@ -99,6 +99,10 @@ const shouldShowAllButton = computed(() => {
     line-height: 2.5rem;
   }
 
+  &__description {
+    margin: 0;
```

Comment:
why is it needed? on onyx.schwarz there is no margin here.
![image](https://github.com/SchwarzIT/onyx/assets/151019977/0f6d6a4b-223c-45cb-8d56-5b13794a82ee)

does not seem to have an effect in 
http://localhost:6006/?path=/docs/roadmap-componentroadmap--docs
if I turn it off either.

Reply:
The "vp-raw" does not work in Storybook so there it will remove the default p margin but when you look at the actual vitepress docs, the margin would be added by the browser so we need to remove it

Reply:
It has an effect for me 🤔

Reply:
weird.. ok, can't hurt to have it ^^ ¯\\_(ツ)\_/¯